test/FunctionalTests/Tests/DataOData/Tests/OData.Common.Tests/PublicApi/PublicApi.bsl

```diff
@@ -7973,7 +7973,11 @@ public class Microsoft.OData.Client.MessageWriterSettingsArgs {
 }
 
 public class Microsoft.OData.Client.QueryOperationResponse : Microsoft.OData.Client.OperationResponse, IEnumerable {
+	long Count  { public virtual get; }
 	Microsoft.OData.Client.DataServiceRequest Query  { public get; }
+	[
+	ObsoleteAttribute(),
+	]

```

Comment:
When adding the obsolete attribute, it is good to add a message to be displayed and whether to classify the "obsoletion" as an error or a warning ie
```[ObsoleteAttribute("This property is obsolete, please use Count", false)]```
and have the attribute in one line.
also please specify the access level of the prop: public, private, internal?

Reply:
@g2mula This file is generated. It's used by the Astoria end to end tests